script/codebuild/build

```
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+# Prints shell input lines as they are read.
+# set -v
+
+# Exit immediately if a command exits with a non-zero status
+set -e
+
+###########
+# INSTALL #
+###########
+
+# Fail if we're not using php71+
+if ! (php --version | grep -q "PHP 7.") ; then
+    echo "You're not using PHP 7.1+, please install it."
+    php --version
+    exit 1
+fi
+
+# copy the certificate from aws parameter store so we can use cosmos-release inside CodeBuild
+# this is a temporal solution until we have an official way to provide CodeBuild with certs
+echo $BBC_CERTIFICATE | base64 --decode > /etc/pki/tls/certs/client.crt
+echo $BBC_CERTIFICATE_KEY | base64 --decode > /etc/pki/tls/private/client.key
+echo $COSMOS_REQUIRED_CA | base64 --decode >> /etc/pki/tls/certs/ca-bundle.crt
+
+if [[ -n "$OAUTH" ]]; then
+    composer config -g github-oauth.github.com $OAUTH
+fi
+# Make sure the reports directory exists
+if [[ ! -d "build/reports" ]]; then
+    mkdir -p build/reports
+fi
+
+composer --no-ansi --no-interaction install
+
+RESULT=0
+
+# Run PHPUnit
+vendor/bin/phpunit --no-coverage || RESULT=1
+
+# Run PHPCS
+vendor/bin/phpcs --report=checkstyle --report-file=build/reports/checkstyle.xml || RESULT=1
```

```suggestion
vendor/bin/phpcs -n || RESULT=1
```
I think we don't want the reports.
In programmes-frontend we have the paralemter `-n` which means "Do not print warnings". Do we want the same here?